tests/unit/transformers/test_numerical.py

```diff
@@ -1391,60 +1391,78 @@ def test__transform(self):
             -0.06969212, -0.06116121, -0.08675394, -0.08675394, -0.07822303,
             0.07374234, 0.05709835, 0.09870834, 0.10703034, 0.05709835
         ])
-        np.testing.assert_allclose(result['continuous'].to_numpy(), expected_continuous, rtol=0.01)
+        np.testing.assert_allclose(result['continuous'].to_numpy(), expected_continuous, atol=0.1)
 
         expected_discrete = np.array([1, 1, 1, 1, 1, 0, 0, 0, 0, 0])
         np.testing.assert_allclose(result['discrete'].to_numpy(), expected_discrete)
 
     def test__reverse_transform_helper(self):
-        """Test '_inverse_transform_continuous' with sigmas != None.
+        """Test '_inverse_transform_helper' with `sigma != None`."""
```

Comment:
Let's use the same format we've been using for unit tests and have a description for `Setup`, `Input` and `Output`.